Python/fileutils.c

```diff
@@ -1275,7 +1277,9 @@ _Py_stat(PyObject *path, struct stat *statbuf)
 static int
 get_inheritable(int fd, int raise)
 {
-#ifdef MS_WINDOWS
+#ifdef MS_WINDOWS_NON_DESKTOP
```

Comment:
Note for reference if turns out we can support `GetHandleInformation()` if `HANDLE_FLAG_INHERIT` is manually defined.

Reply:
added back by manually defining `HANDLE_FLAG_INHERIT`